triagers.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
my only concern is this file

our source of truth is codeowners

the more granular we are, the more error prone we are

did you try to add this info `CODEOWNER`? it is simple text file, yeah not that easy to extract info with `jq` but there are tricks that people use. They could be added as additional comment to CODEOWNER file like:
```
#mdTriagers:@TRohit20,@octonawish-akcodes,@BhaswatiRoy,@VaishnaviNandakumar,@Arya-Gupta,@J0SAL
```

it should be easy to parse file and extract handles with `grep`

Reply:
I can help out if you need help in bash scripting 🤗

Reply:
@derberg @Shurtu-gal Done in the latest commit.